index.html

```diff
@@ -1,10 +1,104 @@
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <title>Tribute Page</title>
 </head>
 <body>
+<div id="main-container">
+    <form id="survey-form" action="">
+        <h1>Survey Form</h1>
+        <div class="container">
+            <div class="row ">
+                <div class="col-sm col-label">
+                    <label id="name-label">Name: </label>
+                </div>
+                <div class="col-sm">
+                    <input type="text" id="name" class="form-control text-field" placeholder="Please enter name" required>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm col-label">
+                    <label id="email-label">Email: </label>
+                </div>
+                <div class="col-sm">
+                    <input type="email" id="email" class="form-control text-field" placeholder="Please enter email address" required>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm col-label">
+                    <label id="number-label">Age: </label>
+                </div>
+                <div class="col-sm">
+                    <input type="number" class="form-control text-field" id="number" min="10" max="60" placeholder="Please enter your age" required> <!--add validation-->
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm col-label">
+                    <label id="city-label">City: </label>
+                </div>
+                <div class="col-sm">
+                    <select id="dropdown" name="City" class="form-control">
+                        <option value="delhi">Delhi</option>
+                        <option value="mumbai">Mumbai</option>
+                        <option value="bangalore">Bangalore</option>
+                        <option value="hyderabad">Hyderabad</option>
+                    </select>
+                </div>
+            </div>
+            <div class="row align-content-start">
+                <div class="col-sm col-label">
+                    <label id="gender-label">Gender: </label>
+                </div>
+                <div class="col-sm form-check form-check-inline">
+                    <input type="radio" name="gender" id="gender1" value="M" class="form-check-input"> <label class="form-check-label" for="gender1">Male</label>&nbsp;
```

Comment:
remove break lines, wrap it in each input and label in div

Reply:
this was for 1 space